### ISSUE TITLE
How did you train it to obtain those binary files?(similarity.pkl and movies_dict.pkl)

### ISSUE THREAD

**Issue by liperrino** [KaProDes/Movie_Pundit#1]
Hi. I am very interested in what you did and i would like to know how you have proceeded to train your model. Can we have access to the source code of the model?

**Comment by KaProDes** [comment]
Absolutely. You can find a link to the [Google Collab](https://colab.research.google.com/drive/148dyD-qc0W6bPDtMAcHifosqFqbdX9SG?usp=sharing) in the [Project Readme](https://github.com/KaProDes/Movie_Pundit/blob/master/readme.md) 

You can download the jupyter notebook and learn more about the model from here : [Jupyter Notebook](https://drive.google.com/file/d/1ZHymcUAIC8lbsDRwPlaqOBTMihuw1Ja8/view?usp=sharing)

I have tried to make the notebook as readable as possible with comments and markdown.
Still, if you have any doubts, please ask in this thread.
